src/app/view/view.component.ts

```diff
@@ -107,29 +115,71 @@ export class ViewComponent implements OnInit, OnDestroy {
 
   private loadQuery(query: Query, name?: string) {
     this.view = {
-      name: name ? `${name} - copy` : '',
+      name: name ? `${name}` : '',
       query: query,
       perspective: null,
       config: {},
     };
   }
 
-  public onSave(name: string) {
+  public onSaveOrClone(name: string) {
+    this.onSave(name, true);
+  }
+
+  public onSave(name: string, clone?: boolean) {
     this.subscriptions.add(
       combineLatest(this.store.select(selectPerspectiveConfig), this.store.select(selectPerspective))
         .pipe(take(1))
         .subscribe(([config, perspective]) => {
           const view: ViewModel = {...this.view, query: this.query, name, config: {[perspective]: config}, perspective};
 
           if (view.code) {
-            this.updateView(view);
+            if (clone) {
+              this.askToCloneView(view);
```

Comment:
Does the fact that you save a view with only name changed means that you are going to clone it? Because your code seems to work this way.

Reply:
Yes, look at this https://lumeer.myjetbrains.com/youtrack/issue/LMR-1265#tab=Time%20Tracking